models/organization/team_user.go

```diff
@@ -47,22 +47,26 @@ type SearchMembersOptions struct {
 func (opts SearchMembersOptions) ToConds() builder.Cond {
 	cond := builder.NewCond()
 	if opts.TeamID > 0 {
-		cond = cond.And(builder.Eq{"team_user.team_id": opts.TeamID})
+		cond = cond.And(builder.Eq{"": opts.TeamID})
 	}
 	return cond
 }
 
 // GetTeamMembers returns all members in given team of organization.
 func GetTeamMembers(ctx context.Context, opts *SearchMembersOptions) ([]*user_model.User, error) {
 	var members []*user_model.User
-	sess := db.GetEngine(ctx).
-		Join("INNER", "team_user", "team_user.uid=user.id").
-		Asc("full_name + name").
-		Where(opts)
+	sess := db.GetEngine(ctx)
+	if opts.TeamID > 0 {
+		sess = sess.In("id",
+			builder.Select("uid").
+				From("team_user").
+				Where(builder.Eq{"team_id = ?": opts.TeamID}),
+		)
+	}
 	if opts.PageSize > 0 && opts.Page > -1 {
 		sess = sess.Limit(opts.PageSize, opts.Page*opts.PageSize)
 	}
-	if err := sess.Find(&members); err != nil {
+	if err := sess.Asc("full_name + name").Find(&members); err != nil {
```

Comment:
Invalid sql, at least invalid for MySQL.

```
mysql> select 'a'+'b';
+---------+
| 'a'+'b' |
+---------+
|       0 |
+---------+
```

Reply:
Hmm..... that's an old bug, so it's fine to leave it here.

Reply:
I changed there to use `CONCAT`. Let's see the CI result.

Reply:
SQLite doesn't support CONCAT.

Could it be `ORDER BY full_name, name` ?